stdlib/printexc.ml

```diff
@@ -69,6 +69,11 @@ let to_string x =
               constructor ^ (fields x) in
   conv !printers
 
+let to_string_opt x =
+  match !printers with
+      [] -> None
```

Comment:
If this stands, please indent as in other `match` cases (so reduce indentation) and include a `|` on the first case.